d2l-outcomes-coa-eval-override.js

```diff
@@ -163,100 +162,116 @@ export class d2lOutcomesCOAEvalOverride extends LocalizeMixin(LitElement) {
 		`;
 	}
 
-	render() {
+	renderElementHeading() {
 		return html`
-		<div class="flex-box">
-			<h3 class="page-heading">Select Overall Achievement</h3>
-		${(!!this._calculationMethod && this._newAssessmentsAdded && this._isOverrideActive)
-		? html`
+			<div class="flex-box">
+				<h3 class="page-heading">Select Overall Achievement</h3>
+				${this.renderCalcButton()}
+			</div>
+		<div style="clear: both;"></div>`;
+	}
+
+	renderCalcButton() {
+		if (this._isCalculationUpdateNeeded()) {
+			return html`
 			<span class="calculate-button-container">
 				<d2l-button-icon id="calculate-button"
 					@click=${this._onCalcButtonClicked}
 					text="${this.localize('recalculateOverallAchievement')}"
 					icon="tier1:calculate">
 				</d2l-button-icon>
-			</span>`
-		: html``}
-		</div>
-
-		<div style="clear: both;"></div>
+			</span>`;
+		}
+		return html``;

```

Comment:
Return null instead of empty html
```suggestion
		return null;
```